content/topics/file-and-directory-naming/python/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - topics/file-and-directory-naming/general
+ready: true
+tags:
+- File and directory naming
+- python
+title: File and directory naming in Python
+---
+
+## How should I name my Python modules and packages?
+
+You should name your files and directories the same way you name Python variables.They should be small letters, and words should be separated by underscores, as that improves readability.
+
+It is important to note that the use of dashes in directory-naming takes preference as opposed to the use of underscores and be sure to keep in mind that all Python files have a .py extension. See below the recommended Python file and directory naming style. 
```

Comment:
This should be at least 2 sentences. And the sentences should make sense. We want them to use underscores, not dashes. This sentence makes it seem like we want dashed instread of underscores.